.github/workflows/build_container_image.yaml

```
@@ -122,3 +131,133 @@ jobs:
+          kubectl get pods -n galaxy -o wide
+
+          echo "=== All pods in galaxy-deps namespace ==="
+          kubectl get pods -n galaxy-deps -o wide
+
+          echo "=== CSI Drivers ==="
+          kubectl get csidriver
+
+          echo "=== Storage Classes ==="
+          kubectl get sc
+
+          echo "=== PVCs in GKM namespace ==="
+          kubectl get pvc -n galaxy
+
+          echo "=== PVCs in galaxy-deps namespace ==="
+          kubectl get pvc -n galaxy-deps
+
+          echo "=== Recent events in GKM namespace ==="
+          kubectl get events -n galaxy --sort-by='.lastTimestamp' | tail -50
+
+          echo "=== Recent events in galaxy-deps namespace ==="
+          kubectl get events -n galaxy-deps --sort-by='.lastTimestamp' | tail -50
+
+          echo "=== Describe pending/failed pods in galaxy namespace ==="
+          for pod in $(kubectl get pods -n galaxy --field-selector=status.phase!=Running,status.phase!=Succeeded -o name 2>/dev/null); do
+            echo "--- Describing $pod ---"
+            kubectl describe -n galaxy $pod
+          done
+
+          echo "=== Describe pending/failed pods in galaxy-deps namespace ==="
+          for pod in $(kubectl get pods -n galaxy-deps --field-selector=status.phase!=Running,status.phase!=Succeeded -o name 2>/dev/null); do
+            echo "--- Describing $pod ---"
+            kubectl describe -n galaxy-deps $pod
+          done
+      - name: Check the version
+        run: |
+          kubectl get svc -n galaxy
+          kubectl describe svc -n galaxy galaxy-nginx
+          address=$(kubectl get svc -n galaxy galaxy-nginx -o jsonpath="http://{.spec.clusterIP}:{.spec.ports[0].port}/galaxy/api/version")
+          echo "Address is $address"
+          appVersion=${{ needs.build.outputs.version }}
+          apiVersion=$(curl $address | jq -r '"\(.version_major).\(.version_minor)"')
+          echo "appVersion: $appVersion"
+          echo "apiVersion: $apiVersion"
+          if [ "$appVersion" != "$apiVersion" ]; then
+            exit 1
+          fi
+
+  pr:
+    name: Create a PR to update the Galaxy Helm chart when a release is tagged
+    runs-on: ubuntu-latest
+    needs: [smoke-test, build]
+    if: startsWith(github.ref, 'refs/tags/')
+    steps:
+      - name: Checkout Galaxy Helm chart
+        uses: actions/checkout@v5
+        with:
+          repository: galaxyproject/galaxy-helm
+          token: ${{ secrets.GITHUB_TOKEN }}
+          persist-credentials: true
+
+      - name: Update Chart.yaml appVersion
+        run: |
+          sed -i "s/^appVersion:.*/appVersion: \"${{ needs.build.outputs.version }}\"/" galaxy/Chart.yaml
+
+      - name: Update values.yaml image.tag
+        run: |
+          sed -i "s/^  tag:.*/  tag: \"${{ needs.build.outputs.tag }}\"/" galaxy/values.yaml
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          token: ${{ secrets.PULL_REQUEST_TOKEN }}
```

Maybe something a bit more targeted like:
```suggestion
          token: ${{ secrets.GALAXY_HELM_PULL_REQUEST_TOKEN }}
```
?